WORKSPACE

```diff
@@ -46,40 +46,4968 @@ load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies", "go_repository")
 ############################################################
 
 go_repository(
-   name = "com_github_pelletier_toml",
-   importpath = "github.com/pelletier/go-toml",
-   sum = "h1:tjENF6MfZAg8e4ZmZTeWaWiT2vXtsoO6+iuOjFhECwM=",
-   version = "v1.9.4"
+    name = "com_github_pelletier_toml",
+    importpath = "github.com/pelletier/go-toml",
+    sum = "h1:tjENF6MfZAg8e4ZmZTeWaWiT2vXtsoO6+iuOjFhECwM=",
+    version = "v1.9.4",
 )
 
 go_repository(
-   name = "com_github_google_go_cmp",
-   importpath = "github.com/google/go-cmp",
-   sum = "h1:BKbKCqvP6I+rmFHt06ZmyQtvB8xAkWdhFyr0ZUNZcxQ=",
-   version = "v0.5.6",
+    name = "com_github_google_go_cmp",
+    importpath = "github.com/google/go-cmp",
+    sum = "h1:81/ik6ipDQS2aGcBfIN5dHDB36BwrStyeAQquSYCV4o=",
+    version = "v0.5.7",
 )
 
 go_repository(
-   name = "com_github_xeipuuv_gojsonschema",
-   importpath = "github.com/xeipuuv/gojsonschema",
-   sum = "h1:LhYJRs+L4fBtjZUfuSZIKGeVu0QRy8e5Xi7D17UxZ74=",
-   version = "v1.2.0",
+    name = "com_github_xeipuuv_gojsonschema",
+    importpath = "github.com/xeipuuv/gojsonschema",
+    sum = "h1:LhYJRs+L4fBtjZUfuSZIKGeVu0QRy8e5Xi7D17UxZ74=",
+    version = "v1.2.0",
 )
 
 # Indirect dependency of com_github_xeipuuv_gojsonschema
 go_repository(
-   name = "com_github_xeipuuv_gojsonreference",
-   importpath = "github.com/xeipuuv/gojsonreference",
-   sum = "h1:EzJWgHovont7NscjpAxXsDA8S8BMYve8Y5+7cuRE7R0=",
-   version = "v0.0.0-20180127040603-bd5ef7bd5415",
+    name = "com_github_xeipuuv_gojsonreference",
+    importpath = "github.com/xeipuuv/gojsonreference",
+    sum = "h1:EzJWgHovont7NscjpAxXsDA8S8BMYve8Y5+7cuRE7R0=",
+    version = "v0.0.0-20180127040603-bd5ef7bd5415",
 )
 
 # Indirect dependency of com_github_xeipuuv_gojsonschema
 go_repository(
-   name = "com_github_xeipuuv_gojsonpointer",
-   importpath = "github.com/xeipuuv/gojsonpointer",
-   sum = "h1:zGWFAtiMcyryUHoUjUJX0/lt1H2+i2Ka2n+D3DImSNo=",
-   version = "v0.0.0-20190905194746-02993c407bfb",
+    name = "com_github_xeipuuv_gojsonpointer",
+    importpath = "github.com/xeipuuv/gojsonpointer",
+    sum = "h1:J9EGpcZtP0E/raorCMxlFGSTBrsSlaDGf3jU/qvAE2c=",
+    version = "v0.0.0-20180127040702-4e3ac2762d5f",
+)
+
+go_repository(
+    name = "com_github_sigstore_rekor",
```

Comment:
Yes, it does seem that we need to include all the transitive dependencies of the go repository in the workspace since it is not already a bazel project. This both how I understand Bazel works (all dependencies need to be included in the bazel rules), and I also tried to build the project with only that single dependency. We went ahead and merged a PR that adds the dependencies in a similar way as this draft. I hope that's OK, but I also don't know of another way around it.

What makes you say `com_github_stretchr_testify` was pulled in from `com_github_xeipuuv_gojsonschema` rather than as a dependency of rekor? I can't tell this by looking at the WORKSPACE file.